colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -294,8 +296,4 @@ extends Controller[O,I](codec, ControllerConfig(config.pendingBufferSize)) with
     }
   }
 
-
-  def idleCheck(period: Duration) {
```

Comment:
So there's still the issue that we're not going to be timing out any sent requests, as that check is definitely up to service layer.  However I don't think we've fully fleshed out what we want the behavior to be when that happens, so we can push that off to another issue.